plugins/ril.c

```diff
@@ -73,7 +73,7 @@
 #define RILD_MAX_CONNECT_RETRIES 5
 #define RILD_CONNECT_RETRY_TIME_S 5
 
-char *RILD_CMD_SOCKET[] = {"/dev/socket/rild", "/dev/socket/rild1"};
+char *RILD_CMD_SOCKET[] = {"/dev/socket/rild", "/dev/socket/rild2"};
```

Comment:
+1 to changing the name. However, this array was introduced to support some qualcomm multi-sim modems (see 2845fef and following commits), so please make sure that plugins/qcom-msim.c still uses rild1 instead of rild2, by overriding in some way this.